From: Rik van Riel <riel@redhat.com>
Date: Tue, 13 Jan 2009 11:37:50 -0500
Subject: [xen] irq: remove superfluous printk
Message-id: 20090113113750.317080d5@bree.surriel.com
O-Subject: [RHEL 5.4 PATCH] remove superfluous printk bz456095
Bugzilla: 456095
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>

When doing live migrate of a system, we take down all the CPUs
through CPU hotplug.  This will trigger the printk, which is
harmless except for the fact that people keep calling support
about it.

Fixes bug 456095

# HG changeset patch
# User Keir Fraser <keir.fraser@citrix.com>
# Date 1211880834 -3600
# Node ID 936f6dd9d49c71c95931c9d8bdf4393cdb9fb42f
# Parent  446c39a967fc0904e2d27ca876c671b414fb5917
x86_64: Remove warning message about 'Breaking affinity for irq'.

32-bit equivalent checked in as:
 http://xenbits.xensource.com/xen-unstable.hg/rev/d24ab5abf857

Signed-off-by: Masaki Kanno <kanno.masaki@jp.fujitsu.com>

diff --git a/arch/x86_64/kernel/irq-xen.c b/arch/x86_64/kernel/irq-xen.c
index 05ddad3..aa77e50 100644
--- a/arch/x86_64/kernel/irq-xen.c
+++ b/arch/x86_64/kernel/irq-xen.c
@@ -149,7 +149,7 @@ void fixup_irqs(cpumask_t map)
 
 		cpus_and(mask, irq_desc[irq].affinity, map);
 		if (any_online_cpu(mask) == NR_CPUS) {
-			printk("Breaking affinity for irq %i\n", irq);
+			/*printk("Breaking affinity for irq %i\n", irq);*/
 			mask = map;
 		}
 		if (irq_desc[irq].chip->set_affinity)
